jetstream/workflows/rerun.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Workflow
+metadata:
+  generateName: jetstream-
+spec:
+  entrypoint: jetstream
+  onExit: export-statistics   # export statistics after all analyses 
+  arguments:
+    parameters:
+    - name: experiment_slug   # set dynamically when workflow gets deployed
+  templates:
+  - name: jetstream
+    parallelism: 5  # run up to 5 containers in parallel at the same time
+    steps:
+    - - name: get-active-experiments
+        template: get-active-experiments
+    - - name: analyse-experiment
+        template: analyse-experiment
+        arguments:
+          parameters:
+          - name: experiment
+            value: "{{item}}"
+        withParam: "{{steps.get-active-experiments.outputs.result}}"  # process active experiments in parallel
+    - - name: export-statistics
+        template: export-statistics # after all analyses are completed, export to GCS
+
+  - name: get-active-experiments
+    container:
+      image: gcr.io/moz-fx-data-experiments/jetstream:latest
+      command: [bin/entrypoint, get-active-experiments, --experiment_slug={{inputs.parameters.experiment_slug}}]
+
+  - name: analyse-experiment
+    inputs:
+      parameters:
+      - name: experiment
+    container:
+      image: gcr.io/moz-fx-data-experiments/jetstream:latest
+      command: [bin/entrypoint, analyse-experiment, "--experiment_config={{inputs.parameters.experiment}}"]
+      resources:
+        requests:
+          memory: 2Gi
```

Comment:
this seems low to me but maybe I'm superstitious -- what's the difference between `requests` and `limits`?

Reply:
Each pod that runs a task will at least get 2 GB of memory but only max. up to 10 GB.